src/app/suraj/Statistics.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+
+
+const Statistics = () => {
+  return (
+    // <div className="container" >
+    // <div className='row row-cols-1 row-cols-md-2 row-cols-lg-3 g-12' style={{display: "flex" , flexDirection:"row", justifyContent:'center',gap:"5rem"}}>
+    //   <div className='card m-3 mt-3 shadow p-3 mb-5 bg-white rounded' style={{border:"2px solid Red" , width: "18rem", height: "300px"}}>
+    //     <div className="card-body text-dark">hello1</div>
+    //   </div>
+    //   <div className='card m-3 mt-3 shadow p-3 mb-5 bg-white rounded' style={{border:"2px solid Red" , width: "18rem", height: "300px" }}>
+    //     <div className="card-body text-dark">hello2</div>
+    //   </div>
+    //   <div className='card m-3 mt-3 shadow p-3 mb-5 bg-white rounded' style={{border:"2px solid Red" , width: "18rem", height: "300px"}}>
+    //     <div className="card-body text-dark">hello3</div>
+    //   </div>
+    // </div> 
+    // <div className='col-12 col-md-6 col-lg-3' style={{display: "flex" , flexDirection:"row",justifyContent:'center',gap:"5rem"}} > 
+    //   <div className='card m-3 mt-3 shadow p-3 mb-5 bg-white rounded' style={{border:"2px solid Red" , width: "18rem", height: "300px"}}>
+    //     <div className="card-body text-dark">hello4</div>
+    //   </div>
+    //   <div className='card m-3 mt-3 shadow p-3 mb-5 bg-white rounded' style={{border:"2px solid Red"  , width: "18rem", height: "300px"}}>
+    //     <div className="card-body text-dark">hello5</div>
+    //   </div>
+    //   <div className='card m-3 mt-3 shadow p-3 mb-5 bg-white rounded' style={{border:"2px solid Red" , width: "18rem", height: "300px" }}>
+    //     <div className="card-body text-dark">hello6</div>
+    //   </div>
+
+    // </div>
+    // </div>
+
+    <div className="container p-5" style={{color:"green"}}>
+      <div className="text-dark lg:text-[20px] md:text-[18px] font-semibold " >
+        <h1 className="mx-16 py-10 mb-3 text-4xl font-normal">Hackodisha 3.0 Statistics</h1>
+        <div className=" grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 m-5" style={{gap:"3rem"}}>
```

Comment:
same here, use tailwind classes. You may ask if you can't figure out which class to use.